stats.go

```diff
@@ -34,3 +34,109 @@ func Median(input []float64) (output float64) {
 
 	return output
 }
+
+// Max returns the maximum value of the sample
+func Max(input []float64) float64 {
+	if len(input) == 0 {
+		return math.NaN()
+	}
+	max := input[0]
+	for i := 1; i < len(input); i++ {
+		if input[i] > max {
+			max = input[i]
+		}
+	}
+	return max
+}
+
+// Min returns the minimum value of the sample
+func Min(input []float64) float64 {
+	if len(input) == 0 {
+		return math.NaN()
+	}
+	min := input[0]
+	for i := 1; i < len(input); i++ {
+		if input[i] < min {
+			min = input[i]
+		}
+	}
+	return min
+}
+
+// Range returns the difference between the largest and smallest values
+func Range(input []float64) float64 {
+	if len(input) < 2 {
+		return math.NaN()
+	}
+	return Max(input) - Min(input)
+}
+
+// sumOfSquaredDifferences returns the sum of the squared differences of each observation from the mean
+func sumOfSquaredDifferences(input []float64) float64 {
+	if len(input) < 2 {
+		return math.NaN()
+	}
+	mean := Mean(input)
+	ssd := 0.0
+	for _, o := range input {
+		ssd += math.Pow(o-mean, 2.0)
+	}
+	return ssd
+}
+
+// StandardDeviationSample returns the standard deviation of the sample
+func StandardDeviationSample(input []float64) float64 {
+	return math.Sqrt(VarianceSample(input))
+}
+
+// VarianceSample returns the variance of the sample
+func VarianceSample(input []float64) float64 {
```

Comment:
Why adding `Sample` at the end? I think `Variance` is enough

Reply:
Ok